xbmc/windowing/win10/WinSystemWin10DX.cpp

```diff
@@ -153,3 +154,41 @@ void CWinSystemWin10DX::UninitHooks()
 void CWinSystemWin10DX::InitHooks(IDXGIOutput* pOutput)
 {
 }
+
+bool CWinSystemWin10DX::IsHDRDisplay()
+{
+  if (CWIN32Util::GetWindowsHDRStatus() != HDR_STATUS::HDR_UNSUPPORTED)
+    return true;
+
+  return false;
```

Comment:
you can simplify this:
```suggestion
return (GetOSHDRStatus() != HDR_STATUS::HDR_UNSUPPORTED)
```

Reply:
I like the simplification, however I keep `GetWindowsHDRStatus` because it's more direct: `GetOSHDRStatus` calls `GetWindowsHDRStatus` on Windows.